x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate.tsx

```diff
@@ -0,0 +1,620 @@
+              {
+                defaultMessage: 'Another feature not listed here',
+              }
+            )}
+          </p>
+        </EuiText>
+      </Fragment>
+    ),
+    inputDisplay: i18n.translate(
+      `${baseTranslatePrefix}.other.inputDisplay`,
+      {
+        defaultMessage: 'Other',
+      }
+    ),
+    value: 'other',
+  });
+
+  return featureList;
+};
+
+const participateInUXLabsChoice = {
+  no: { choice: 'no', value: false },
+  yes: { choice: 'yes', value: true },
+};
+
+const EducationPanel: React.FC<{ featureContent: string }> = ({ featureContent }) => {
+  const feature = getFeature(featureContent);
+  const { setFeaturesOther } = useActions(AppSearchGateLogic);
+  if (feature) {
+    return (
+      <EuiPanel hasShadow={false}>
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <EuiIcon type="logoElastic" />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup direction="column" gutterSize="xs">
+              <EuiFlexItem>
+                <EuiText>
+                  <h5>
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.workplaceSearch.gateForm.educationalPanel.title',
+                      {
+                        defaultMessage: 'Elasticsearch native equivalent',
+                      }
+                    )}
+                  </h5>
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiText color="subdued" size="s">
+                  <p>
+                    {i18n.translate(
+                      'xpack.enterpriseSearch.workplaceSearch.gateForm.educationalPanel.subTitle',
+                      {
+                        defaultMessage: 'Based on your selection we recommend:',
+                      }
+                    )}
+                  </p>
+                </EuiText>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiSpacer />
+
+        <EuiCallOut title={feature.title} color="success" iconType="checkInCircleFilled">
+          <p>{feature.panelText}</p>
+          <EuiFlexGroup gutterSize="m" wrap alignItems="baseline">
+            {feature.actionLink !== undefined && feature.actionLabel !== undefined && (
+              <EuiFlexItem grow={false}>
+                <EuiButton href={feature.actionLink} target="_blank" iconType="sortRight" iconSide="right">
+                  {feature.actionLabel}
+                </EuiButton>
+              </EuiFlexItem>
+            )}
+
+            {feature.learnMore != undefined && (
+              <EuiFlexItem grow={false}>
+                <EuiLink href={feature.learnMore} target="_blank">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.workplaceSearch.gateForm.educationalPanel.learnMore',
+                    {
+                      defaultMessage: 'Learn More',
```

Comment:
Should this be `Learn more`?

Reply:
agree with them needing to be `Learn more`